mix.exs

```diff
@@ -27,7 +27,7 @@ defmodule HeroiconsElixir.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:phoenix_live_view, "~> 0.18.2"},
+      {:phoenix_live_view, "~> 0.19"},
```

Comment:
Could we just do `>= 0.18.2` in order to make this not backwards incompatible?

Reply:
I'm not sure I follow...

I'd like someone on live view 0.18 to be able to use newer versions of this library without upgrading live view.

Reply:
On second thought, I think your right, your solution is really practical

Reply:
Added the suggested changes here - @mveytsman  @dovadi changed version back to the running 0.5.2 as well FYI